src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -91,14 +97,20 @@
 					<Hint v-else :hint="t('spreed', 'No search results')" />
 				</template>
 			</div>
+			<Button v-if="!preventFindingUnread && unreadBottom > 0"
+				class="unread-mention-button unread-mention-button__bottom"
+				type="primary"
+				@click="scrollBottomUnread">
+				{{ unreadBottom }} more mentions
```

Comment:
Needs to be translated.
Best is to do that in a computed:
```js
		unreadBottomLabel() {
			return n('spreed', '%n unread mention', '%n unread mentions', this.unreadBottom)
		},
```

n is for plurals, maybe you read https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#javascript quickly :)

Reply:
After checking the mobile apps, we don't show a count there, so if that makes it a bit more performant or something we can also skip it and just go with:

```suggestion
				{{ t('spreed', 'Unread mentions') }}
```